### ISSUE TITLE
Installation not correct

### ISSUE THREAD

**Issue by The1nstructor** [thefapinstructor/fapinstructor#15]
### Installation not working

EDIT:
befor that `npm` shows the following warning:
```
npm install || yarn
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. 
You must install peer dependencies yourself.

audited 15907 packages in 8.296s
found 0 vulnerabilities
```
after `npm start`
The error displayed is:
**Failed to compile**
```
./src/index.js
Module not found: `/.../.Inst/fapinstructor/src/components/Root.js` does not match the corresponding path on disk `Components`.
```

Error occured on `macOS 10.13.5` in Chrome

**Comment by ArtisticMenase** [comment]
I also ran into the Components/components and Containers/containers issue in windows, so it isn't a  MacOS only issue.

**Comment by The1nstructor** [comment]
I strongly believe 80f34d3974a6ce905ff5e04ad36aa4e55d935db9 fixed the following: 
The due to the duplicate directories either upper or lowercase not found folders have been removed.

I don't know whether also all dependencies are fulfilled (ajv). But from this point of view this closes #15
